src/features/CareViewSummary/components/CareViewSummary.jsx

```diff
@@ -1,24 +1,119 @@
+    callbacks.setIsLoading(true);
+    const response = await fetchWardSummary(selectedWard.value);
+    if (response.status === 200) {
+      setWardSummary(response.data);
+    } else {
+      setWardSummary({});
+    }
+    callbacks.setIsLoading(false);
+  };
+  useEffect(() => {
+    getWardList();
+  }, []);
+  useEffect(() => {
+    if (!_.isEmpty(selectedWard)) {
+      getWardSummary();
+    }
+  }, [selectedWard]);
   return (
     <div className="care-view-summary">
       <Dropdown
         id="default"
         label="Dropdown menu options"
-        items={["General Ward (A)", "General Ward (B)", "General Ward (C)"]}
-        itemToString={(item) => (item ? item : "")}
-        selectedItem={"General Ward (A)"}
+        options={options}
+        onChange={(e) => {
+          if (e) {
+            setSelectedWard(e);
+          }
+        }}
+        selectedValue={selectedWard}
+        titleText={""}
       />
       <div className="summary-tiles">
-        <Tile className="summary-tile">Total patients</Tile>
-        <Tile className="summary-tile">My patients</Tile>
-        <Tile className="summary-tile">New patients</Tile>
-        <Tile className="summary-tile">Scheduled for OT</Tile>
-        <Tile className="summary-tile">To be discharged</Tile>
+        <Tile className="summary-tile">
+          <span className={"heading-text"}>
+            <FormattedMessage
+              id={"TOTAL_PATIENTS"}
+              defaultMessage={"Total patients"}
+            />
+          </span>
+          <span className={"value-text"}>{wardSummary.totalPatients || 0}</span>
+        </Tile>
+        <Tile className="summary-tile">
+          <span className={"heading-text"}>
+            <FormattedMessage
+              id={"MY_PATIENTS"}
+              defaultMessage={"My patients"}
+            />
+          </span>
+          <span className={"value-text"}>{wardSummary.myPatients || 0}</span>
+        </Tile>
+        <Tile className="summary-tile">
+          <span className={"heading-text"}>
+            <FormattedMessage
+              id={"NEW_PATIENTS"}
+              defaultMessage={"New patients"}
+            />
+          </span>
+          <span className={"value-text"}>{wardSummary.newPatients || 0}</span>
+        </Tile>
+        <Tile className="summary-tile">
+          <span className={"heading-text"}>
+            <FormattedMessage
+              id={"SCHEDULED_FOR_OT"}
+              defaultMessage={"Scheduled for OT"}
+            />
+          </span>
+          <span className={"value-text"}>
+            {wardSummary.scheduledForOT || 0}
+          </span>
+        </Tile>
+        <Tile className="summary-tile">
+          <span className={"heading-text"}>
+            <FormattedMessage
+              id={"TO_BE_DISCHARGED"}
+              defaultMessage={"To be discharged"}
+            />
+          </span>
+          <span className={"value-text"}>
+            {wardSummary.toBeDischarged || 0}
```

Comment:
do we need to show all these count as these are not implemented?

Reply:
Will be removing them in the next story card ( Ability to make page responsive) as it needs some confirmation on the alignments when removed.